alerts.json

```diff
@@ -0,0 +1,41 @@
+ {
+  "JANUSGRAPH": {
+    "JANUSGRAPH_SERVER": [
+      {
+        "name": "janusgraph",
+        "label": "janusgraph server",
+        "description": "This host-level alert is triggered if the JanusGraph Server Instance is unreachable.",
+        "interval": 1,
+        "scope": "ANY",
+        "source": {
+          "type": "SCRIPT",
+	  "path": "JANUSGRAPH/package/alerts/alert_check_janusgraph_server.py",
+	  "parameters": [
+            {
+              "name": "janusgraph.run.dir",
+              "display_name": "Run Directory",
+              "value": "{{janusgraph-env/janusgraph_pid_dir}}",
+              "type": "STRING",
+              "description": "The directory where the JanusGraph server process places its PID files."
+            },
+            {
+              "name": "janusgraph.install.dir",
+              "display_name": "Janusgraph Install Directory",
```

Comment:
In all files, use the correct the capitalization for `JanusGraph` instead of `Janusgraph`.